src/pcp/types.rs

```diff
@@ -245,23 +245,26 @@ mod tests {
     #[test]
     fn test_count() {
         // Test PCP on valid input.
-        pcp_validity_test(
+        let res = pcp_validity_test(
             &Count::<TestField>::new(1).unwrap(),
             &ValidityTestCase {
                 expect_valid: true,
                 expected_proof_len: 9,
             },
         );
-        pcp_validity_test(
+        assert!(res.is_ok(), "error: {:?}", res.unwrap_err());
```

Comment:
You can just do `pcp_validity_test(...).unwrap();` and if `pcp_validity_test` returns an error, `unwrap()` will print it.

Reply:
Good call! Will fix when this PR is next up.

Reply:
Done.